includes/Modules/Analytics/Advanced_Tracking.php

```diff
@@ -79,14 +97,25 @@ public function register() {
 			function() {
 				$this->set_up_advanced_tracking();
 			},
-			11
+			15
 		);
 		add_filter(
 			'googlesitekit_amp_gtag_opt',
 			function( $gtag_amp_opt ) {
 				return $this->set_up_advanced_tracking_amp( $gtag_amp_opt );
 			}
 		);
+		add_action(
+			'wp_footer',
+			function() {
+				if ( null === $this->measurement_code_injector ) {
+					return;
+				}
+				$this->configure_events();
+				$this->measurement_code_injector->inject_event_tracking( $this->event_configurations );
+			},
+			15
+		);
```

Comment:
Let's not add this action hook here: It should be added within the `set_up_advanced_tracking` method instead, since otherwise it'll unconditionally run, even in AMP, where this would cause invalid output. Since in `set_up_advanced_tracking` we check whether the `gtag` script is enqueued, from there on we can safely assume that all requirements are fulfilled and add the `wp_footer` action from there.